tests.sh

```diff
@@ -296,5 +296,23 @@ test_eval 'weekyear("2027-12-27")' '52'
 test_eval 'weekyear("2027-12-31")' '52'
 test_eval 'weekyear("2028-01-01")' '53'
 
+# weekday with date
+test_eval 'weekday("2021-03-21")' '0'
+test_eval 'weekday("2016-03-21")' '1'
+test_eval 'weekday("2017-03-21")' '2'
+test_eval 'weekday("2018-03-21")' '3'
+test_eval 'weekday("2019-03-21")' '4'
+
+# weekday with date_time
+test_eval 'weekday("2014-03-21T21:03")' '5'
+test_eval 'weekday("2015-03-21T21:03")' '6'
+
+# weekday with date and locale'
+test_eval 'weekday("2021-03-21", "nb")' '"Søndag"'
+test_eval 'weekday("2016-03-21", "zh-CN")' '"星期一"'
+test_eval 'weekday("2017-03-21", "fr-FR")' '"Mardi"'
+test_eval 'weekday("2018-03-21", "th-TH")' '"วันพุธ"'
+test_eval 'weekday("2019-03-21", "pt-BR")' '"Quinta-feira"'
```

Comment:
missing tests with unknow locale and invalid date

Reply:
will add!

Reply:
okay, it seems like this tests.sh does not test for thrown errors. i think tests like these are added on the parsec gem, [like this](https://github.com/oxeanbits/parsec/blob/3701864b947448754e3b7e3978cdd798fb319936/test/test_parsec.rb#L179)

Reply:
@niltonvasques tests for thrown errors are done in parsec layer, can we merge this and proceed to parsec PR?

Reply:
It is possible guys @EduardoRSeifert and @Marinofull, to catch crashs on bash: See

https://medium.com/@dirk.avery/the-bash-trap-trap-ce6083f36700


But looking parsec code, seems that it is also possible to get the syntax error:

https://github.com/oxeanbits/parsec/blob/3701864b947448754e3b7e3978cdd798fb319936/lib/parsec.rb#L81-L86

Reply:
Yes, we have tests in the parsec gem that check for syntax error, as i said in my previous comment

Reply:
Thats not the reason that I posted the parsec link here. In my link you can see that is possible to collect the syntax error from the libnativemath output, thus you can also write tests here on the bash.

Reply:
In bash you can also catch crashs/fatals as you can see in the first link.